internal/txtartest/vanilla/vanilla_txtar.go

```diff
@@ -280,24 +285,31 @@ func formatVanillaNode(t *testing.T, n ast.Node) []byte {
 // RawInstances returns the instances represented by this .txtar file. The
 // returned instances are not checked for errors.
 func (t *Test) RawInstances(args ...string) []*build.Instance {
-	return LoadVanilla(t.Archive, t.Dir, args...)
+	return LoadVanilla(t.ThemaFS, t.Archive, args...)
 }
 
 // LoadVanilla loads the instances of a txtar file. By default, it only loads
-// files in the root directory. Relative files in the archive are given an
-// absolute location by prefixing it with dir.
-func LoadVanilla(a *txtar.Archive, dir string, args ...string) []*build.Instance {
+// files in the root directory.
+func LoadVanilla(themaFS fs.FS, a *txtar.Archive, args ...string) []*build.Instance {
```

Comment:
**Note:** we changed the way CUE files are loaded for tests.

Errors include an absolute path to the file that caused it (both for the input data and the schema), implying that running the tests on a different machine would produce a different error message.
To work around this feature, we decided to load CUE files in a way that exclusively relies on a virtual filesystem.

Reply:
Excellent!